README.md

```diff
@@ -122,6 +122,82 @@ With [django-csp](https://django-csp.readthedocs.io/en/latest/nonce.html#middlew
 {% block script_attributes %}nonce="{{request.csp_nonce}}"{% endblock %}
 ```
 
+## ASGI support
+
+Async views with an ASGI server is also supported.
+
+```python
+import asyncio
+
+from suspense.shortcuts import async_render
+
+# app/views.py
+async def view(request):
+    async def obj():
+
+        await asyncio.sleep(1)
+        return range(10)
+
+    return async_render(request, 'template.html', {'obj': obj()})
+```
+
+Suspense will wait for any awaitable object to finish before rendering the suspense tags.
+
+### Specify which awaitable to wait for
+
+If you have multiple suspense blocks with different awaitable, you can specify which awaitable to wait for or each suspense block will await everything.
+
+Ex: `{% suspense obj obj2 %}`
+
+```jinja
+{% load suspense %}
+
+<ul>
+    {% suspense obj %}
+        {% fallback %}
+            <li class="skeleton">Loading ... </li>
+        {% endfallback %}
+
+        {% for data in obj %}
+            <li>{{ data }}</li>
+        {% endfor %}
+    {% endsuspense %}
```

Comment:
It's not entirely obvious what it's for. Inside the block, the `obj` object already needs deferred processing. 

I think if multiple `suspense` blocks are specified we should process them separately, asynchronously. They should not be linked to each other. So for example one block will work in 3 seconds, another 5 seconds.

Reply:
Each suspense block receive the whole context that can containing multiple async tasks. 

I need a way per suspense block to tell what it has to wait for., we do not want every suspense block to wait for everything.

Specifying the variable explicitly would not be necessary if we could find a way to determine what are all the variables used within the suspense.

How it is done actually, each block will display right when they are done only if the appropriate variables to waiting for are assigned properly.